<?xml version="1.0"?>
<!-- vim: set fenc=utf-8 et sts=4 ft=xml : -->
<launch>

    <node
        name="navigation_audio"
        machine="navigate"
        pkg="sound_play"
        type="soundplay_node.py"
        ns="/io/audio">
        <remap from="robotsound" to="navigate_soundplay"/>
    </node>

    <node
        name="search_audio"
        machine="search"
        pkg="sound_play"
        type="soundplay_node.py"
        ns="/io/audio">
        <remap from="robotsound" to="search_soundplay"/>
    </node>

    <!--The nodes below queue and handle announcements from other
        nodes.  They send the announcements to the nodes above-->

    <node
        name="navigate_announcer"
        machine="navigate"
        pkg="samplereturn"
        type="announcer.py"
        ns="/io/audio">
        <remap from="audio_in" to="navigate"/>
        <remap from="audio_out" to="navigate_soundplay"/>
    </node>

    <node
        name="search_announcer"
        machine="search"
        pkg="samplereturn"
        type="announcer.py"
        ns="/io/audio">
        <remap from="audio_in" to="search"/>
        <remap from="audio_out" to="search_soundplay"/>
    </node>

    <node
        name="voltage_announcer"
        machine="navigate"
        pkg="samplereturn"
        type="announce_voltage.py"
        ns="/io">
        <param name="lowVoltageLimit" type="double" value="49"/>
        <remap from="audio_search" to="audio/search"/>
    </node>
</launch>
